modules/op-installing-tkn-on-linux.adoc

```diff
@@ -10,7 +10,13 @@ For Linux distributions, you can download the CLI directly as a `tar.gz` archive
 
 .Procedure
 
-. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.13.1/tkn-linux-amd64-0.13.1.tar.gz[CLI].
+. Download the relevant CLI.
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.13.1/tkn-linux-amd64-0.13.1.tar.gz[Linux]
```

Comment:
should we specify the architecture? (something like x86_64 or amd64)

Reply:
@ppitonak @Preeticp How about: 

Download the relevant CLI.

-     Linux (x86, amd64) 
-     Linux on IBM Z and LinuxONE (s390x)
-     Linux on IBM Power Systems (ppc64le)

Reply:
x86 usually refers to 32-bit architecture which is incompatible, it should be `x86_64`

Reply:
+1 this makes sense

Reply:
@Preeticp Added the architectures